bench-scripts/common_util.sh

```diff
@@ -10,6 +10,12 @@
 
 set -x
 
+INSTALL_ROOT=${BENCH_INSTALL_ROOT:-"/tmp/bench.binaries"}
```

Comment:
why this change is needed? the file is supposed to be included/soruced by other scripts here. those variables should be defined/provided by the script which includes common_util.sh. so IMO doing it here is pointless.